401 Unauthorized error trying to build db*CODECOP v5.0.1

Hi,
I'm trying to build db*CODECOP v5.0.1 using the documentation on the following page: https://github.com/Trivadis/plsql-cop-validators?tab=readme-ov-file, section "How To Build". In step 5, I'm getting the following error messages:

**$ mvn -Dtvdcc.basedir=C:\dbCODECOPv501\tvdcc-5.0.1 clean package**

```
[WARNING] org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xmlfailed to transfer from https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of plsqlcop has elapsed or updates are forced. Original error: Could not transfer metadata org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml, status: 401 Unauthorized
[WARNING] org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xmlfailed to transfer from https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of plsqlcop has elapsed or updates are forced. Original error: Could not transfer metadata org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml, status: 401 Unauthorized
[WARNING] org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xmlfailed to transfer from https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of plsqlcop has elapsed or updates are forced. Original error: Could not transfer metadata org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml, status: 401 Unauthorized
[WARNING] org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xmlfailed to transfer from https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of plsqlcop has elapsed or updates are forced. Original error: Could not transfer metadata org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml, status: 401 Unauthorized
Downloading from plsqlcop: https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/com/google/code/gson/gson/maven-metadata.xml
[WARNING] Could not transfer metadata com.google.code.gson:gson/maven-metadata.xml from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/com/google/code/gson/gson/maven-metadata.xml, status: 401 Unauthorized
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.057 s
[INFO] Finished at: 2024-04-30T17:09:45+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project sonar-plsql-cop-custom-validators-plugin: Could not resolve dependencies for project trivadis.tvdcc:sonar-plsql-cop-custom-validators-plugin:sonar-plugin:5.0.2-SNAPSHOT: Failed to collect dependencies at trivadis.tvdcc:tvdcc:jar:5.0.1: Failed to read artifact descriptor for trivadis.tvdcc:tvdcc:jar:5.0.1: Could not transfer artifact trivadis.tvdcc:tvdcc:pom:5.0.1 from/to plsqlcop (https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1): authentication failed for https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1/trivadis/tvdcc/tvdcc/5.0.1/tvdcc-5.0.1.pom, status: 401 Unauthorized -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Any idea how I can solve this?

Thanks
Matthias

--- a/README.md
+++ b/README.md
@@ -269,7 +269,10 @@ This way you can deal with an unbound number of validators without comproming th
 
 3. Install the required db* CODECOP libraries
 
-   These libraries are not available in public Maven repositories into your local Maven repository. 
+   These libraries are not available in public Maven repositories. However, they are part
+   of the db* CODECOP Command Line installed in step 1. You need to install these libraries
+   into your local Maven repository.
+
    Open a terminal window in the cop-validators root folder and run Run the following shell script:
 
       ./install_tvdcc_libs.sh
@@ -282,9 +285,7 @@ This way you can deal with an unbound number of validators without comproming th
 
    Open a terminal window in the cop-validators root folder and run the Maven build by the following command
 
-		mvn -Dtvdcc.basedir=/usr/local/bin/tvdcc clean package
-
-	Amend the parameter `tvdcc.basedir` to match your `TVDCC_HOME` directory. This folder is used to reference db* CODECOP jar files which are not available in public Maven repositories
+		mvn clean package
 
 ## License
 

--- a/install_tvdcc_libs.sh
+++ b/install_tvdcc_libs.sh
@@ -15,8 +15,8 @@ if ! test -f "${TVDCC_DIR}/tvdcc.jar"; then
 fi
 
 # define versions according usage in pom.xml
-TVDCC_VERSION="5.0.0-SNAPSHOT"
-PLSQL_VERSION="5.0.0-SNAPSHOT"
+TVDCC_VERSION="5.0.1"
+PLSQL_VERSION="5.0.1"
 
 # install JAR files into local Maven repository, these libs are not available in public Maven repositories
 mvn install:install-file -Dfile=$TVDCC_DIR/tvdcc.jar -DgeneratePom=true \

--- a/pom.xml
+++ b/pom.xml
@@ -114,7 +114,7 @@
             <!-- contains PLSQLValidator -->
             <groupId>trivadis.oracle</groupId>
             <artifactId>plsql</artifactId>
-            <version>5.0.0</version>
+            <version>5.0.1</version>
             <scope>provided</scope>
         </dependency>
 		<dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -36,18 +36,6 @@
 					</plugin>
 				</plugins>
 			</build>
-           <repositories>
-                <repository>
-                    <id>plsqlcop</id>
-                    <url>https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1</url>
-                    <releases>
-                        <enabled>true</enabled>
-                    </releases>
-                    <snapshots>
-                        <enabled>true</enabled>
-                    </snapshots>
-                </repository>
-            </repositories>
 		</profile>
 		<profile>
 			<id>idea</id>
@@ -72,18 +60,6 @@
 					</plugin>
 				</plugins>
 			</build>
-           <repositories>
-                <repository>
-                    <id>plsqlcop</id>
-                    <url>https://pkgs.dev.azure.com/tvdvsts/2001.5038513/_packaging/plsqlcop/maven/v1</url>
-                    <releases>
-                        <enabled>true</enabled>
-                    </releases>
-                    <snapshots>
-                        <enabled>true</enabled>
-                    </snapshots>
-                </repository>
-            </repositories>
 		</profile>
 		<profile>
 			<activation>

